src/lib/airtable/request.js

```diff
@@ -186,6 +187,21 @@ export const createManyFinancialSummaries = async (records) => {
   return Promise.all(createPromises);
 };
 
+// NONGENERATED: Create a product (inventory type) and add it to Redux
+export const createNewProduct = async (product) => {
```

Comment:
Can we just modify the `createProduct` function so that we can use the `createProduct` function instead of needing to use `createNewProduct`? Would be confusing to incoming devs.

I was thinking we could modify `createProduct` like the following

```
export const createProduct = async (record) => {
  let productId = '';
  try {
  ...
  productId = await createRecord(Table.Products, record);
  } catch {
  ...
  productId = generateOfflineId()
  }
  ...
  addProductToRedux(...)
  return product.id;
```

so basically what you have except instead of `await createProduct(product)` you do all this logic in `createProduct` and do a `createRecord(Tables.Products, record)` call instead